fuse/plugins/micro_physics/lineage_cluster.py

```diff
@@ -295,10 +332,20 @@ def num_there(s):
     return any(i.isdigit() for i in s)
 
 
-def classify_lineage(particle_interaction):
+def classify_lineage(particle_interaction, secondaries=None):
     """Function to classify a new lineage based on the particle and its parent
     information."""
 
+    # Check if we passed secondaries
+    if secondaries is not None:
+        for secondary in secondaries:
+            # there is a seconday that has type gamma and edproc phot,
+            # we classify this lineage as beta because it is most likely a compton scattering
```

Comment:
This comment is also maybe outdated as the code does not check for gamma and phot edproc?

Reply:
Yes, also from previous version of the code. Thanks!